app/views/about/index.en.html.erb

```diff
@@ -2,18 +2,20 @@
 
 <div class="grid-row">
```

Comment:
We can get rid of this file, it was a test. The real index page is in `views/home/index`

Reply:
Ok great :D I'll remove it!